src/features/CreationForm/ui/Message/Message.tsx

```diff
@@ -0,0 +1,51 @@
+import { Button, Flex } from '@mantine/core';
+import { IconCheck, IconX } from '@tabler/icons-react';
+import clsx from 'clsx';
+import { FC, useCallback } from 'react';
+import { useNavigate } from 'react-router-dom';
+import styles from './Message.module.scss';
+
+interface Props {
+    result: string;
+    onClose?: () => void;
+}
+
+export const Message: FC<Props> = (props) => {
+    const { result, onClose } = props;
+
+    const navigate = useNavigate();
+
+    const goToMain = useCallback(() => {
+        navigate('/');
+    }, [navigate]);
+
+    const icon =
+        result === 'success' ? (
+            <IconCheck className={styles.icon} />
+        ) : (
+            <IconX className={styles.icon} />
+        );
+
+    const actionButton =
+        result === 'success' ? (
+            <Button variant="filled" onClick={goToMain}>
+                На главную
+            </Button>
+        ) : (
+            <Button variant="filled" onClick={onClose}>
+                Закрыть
+            </Button>
+        );
```

Comment:
try to simplify this.
can create a function that does goToMain or onClose depending on the result?